benchmarking/commons/benchmark_definitions/yahpo.py

```diff
@@ -199,21 +226,133 @@ def yahpo_iaml_xgboost_benchmark(dataset_name, metric, mode="max"):
         blackbox_name="yahpo-iaml_xgboost",
         dataset_name=dataset_name,
         max_resource_attr="max_resources",
+        surrogate_kwargs=surrogate_kwargs,
     )
 
 
-yahpo_iaml_xgboost_benchmark_definitions = {
-    f"yahpo-iaml_xgboost_{metric}-{name}": yahpo_iaml_xgboost_benchmark(
-        name, metric, mode
-    )
-    for name in yahpo_iaml_instances
-    for metric, mode in yahpo_iaml_metrics
+yahpo_iaml_benchmark = {
+    "rpart": yahpo_iaml_rpart_benchmark,
+    "glmnet": yahpo_iaml_glmnet_benchmark,
+    "ranger": yahpo_iaml_ranger_benchmark,
+    "xgboost": yahpo_iaml_xgboost_benchmark,
 }
 
 
-def yahpo_rbv2_rpart_benchmark(dataset_name, metric, mode="max"):
+yahpo_iaml_instances = ["40981", "41146", "1489", "1067"]
+
+
+yahpo_iaml_selected_instances = {
+    "rpart": {
+        "f1": ["41146", "1489", "1067"],
+        "auc": ["41146", "1489", "1067"],
+    },
+    "glmnet": {
+        "f1": ["1489", "1067"],
+        "auc": ["1489"],
+    },
+    "ranger": {
+        "f1": ["41146", "1489", "1067"],
+        "auc": ["41146", "1489"],
+    },
+    "xgboost": {
+        "f1": ["40981", "1489"],
+        "auc": ["40981", "1489"],
+    },
+}
```

Comment:
NIT: I think it might be easier to store in a table (in resources) with 
```
benchmark_ID, method, metric, selected
41146       , rpart , f1    , True
41146       , rpart , auc   , True
(...)

```

Reply:
Again, this is not data automatically generated, but selected more or less by hand.